uat/features/mollom.feature

```diff
@@ -15,7 +16,8 @@ Feature: Mollom
     Then I should see "The form protection has been added"
 
 # This will fail until we figure out a way to pass private key
+  @api @dev @destructive
```

Comment:
Well I think this mollom feature is over a year old and I haven't fixed it yet...how's that for lazy?